key-manager/src/main/scala/org/bitcoins/keymanager/bip39/BIP39KeyManager.scala

```diff
@@ -62,6 +63,9 @@ case class BIP39KeyManager(
 object BIP39KeyManager extends BIP39KeyManagerCreateApi with BitcoinSLogger {
   val badPassphrase = AesPassword.fromString("changeMe").get
 
+  private def getCurrentTimeUTC: Long =
```

Comment:
Let's move this to `BitcoinSUtil` if it doesn't exist there yet, or some sort of `TimeUtil` object in core.

Reply:
Also this should probably documented somewhere that we use `creationTime` to be seconds